install.sh

```
@@ -161,6 +161,88 @@ backup_existing_config() {
+        response="${CI_RESPONSE:-k}"
+    fi
+    
+    # Convert to lowercase for compatibility (macOS bash 3.2)
+    response=$(echo "$response" | tr '[:upper:]' '[:lower:]')
+    case "$response" in
+        r|replace)
+            warn "Replacing existing settings.json"
+            cp "$source_file" "$target_file"
+            ;;
+        b|backup)
+            local backup_name="${target_file}.backup-$(date +%Y%m%d-%H%M%S)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)